charters/Proposed-WG-WebIdentityCredentials.md

```diff
@@ -133,6 +136,14 @@ The TAG develops general design principles that will guide the work of
 this Working Group. The TAG might provide input and guidance on specific
 aspects of the work.
 
+[Web Authentication (WebAuthn) Working Group](https://www.w3.org/groups/wg/webauthn/)
+
+While we are not developing an authentication mechanism, it still must work in conjunction with existing authentication mechanisms. The WebAuthn Working Group may provide input and guidance for this requirement.
+
```

Comment:
It's worth noting that the [WebID CG](https://www.w3.org/community/webid/) is aiming to charter a WebID WG in the near future, so it's good that this group's proposed name is changing from _Web Identity Credentials_ to _Federated Identity_. Also, the WebID CG/WG work should be kept in mind as complementary and/or conflicting and/or redundant, etc., depending on overall timing....